content/docs/start/visualization.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Get Started: Visualization with Plots'
+---
+
+# Get Started: Visualization with Plots
```

Comment:
I'm not sure the frontmatter is needed when it matches the H1 actually. I forgot what the use and criteria was here...

Reply:
Other start pages are similar. Keeping this.